### ISSUE TITLE
Complete UX rubric

### ISSUE THREAD

**Issue by TeriDibbits** [paprika-spice/web-app#18]

**Comment by scallsen** [comment]
## DESIGN & RESEARCH COMPONENT

| | 0 points | 1 points | 2 points | 3 points |
| --- | --- | --- | --- | --- |
| **Utilization of Github tools each week** | Did not use Github tools | Sometimes used Github tools | Used Github tools every week somewhat correctly for most tasks | Used Github tools every week correctly for all tasks |
| **Target Audience** | There was no defined target audience defined | The target audience defined somewhat matched the product's goals | The target audience mostly matched the product's goals | The target audience correctly matches the product's goals |
| **Competitive Analysis** | No research was done for competitors | Very little research was done for competitors | Some research was done, and some used in the conceptual development phase | Competitors were researched and considered in the conceptual development phase |
| **North Star - End Goal** | The end goal was not considered, and the application either solved another goal entirely or accomplished nothing at all. | The end product met a few of the goals defined in the development phase | The end product met most of the goals defined in the development phase | The end product met all the goals defined in the conceptual development phase |
| **External Feedback** | There was no user testing performed | 1 or 2 external sources were interviewed/used in user testing | 3 or 4 external sources were interviewed/used in user testing | At least 5 or more external sources were interviewed/used in user testing |
| **Teamwork** | What team? | Some teamwork happened but it fell apart after a short time.  | Communication was something, work was performed ok together | Worked great together. Communicated often and made decisions together. Everyone took on the workload they were assigned. |
| **Quality of wireframes** | Wireframes were not present / such low quality they are unusable | Wireframes were rough and not useful for further development | Wireframes were mostly complete and somewhat organized; usable for further development  | Wireframes are effective, clean, and communicate the website's goals. |
| **Quality of prototype** | Prototype is poorly executed and incomplete | Prototype is somewhat organized and complete | Prototype is mostly organized and complete | Prototype looks cohesive, organized, and includes enough detail |

**Comment by scallsen** [comment]
**Utilization of Github tools each week – 2.5**
We used all the tools we needed, but some members may have incorrectly completed their portion, but later learned for the next time we completed it.

**Target Audience – 3**
Our target audience's needs  - students looking to book a room - matches the goal of our web app. Confirmed after user testing.

**Competitive Analysis – 3**
We researched 9 competitors in our research (from different devices and specialties) and incorporated information from that in further stages of the UX design process.

**North Star / End Goal – 3**
Our app definitely meets the goals of the end user, even after user testing, the changes we have to make are layout based changes, not corrections to the core goal of the app

**External Feedback – 3**
We met with 5 different people and got them to test our app and provide feedback.

**Teamwork – 3**
We communicate mostly in class and assign tasks during this time. We talk on Slack when needed, Teri sends a weekly reminder to make sure we are on track. 

**Quality of wireframes – 2**
Wireframes were pretty thorough but led to a bit of confusion and were missing how the screens linked.

**Quality of prototype – 2**
Through user testing, we found that we could have added more detail to our existing screens, and more screens overall to make it more clear as to what is needed.
